### PR TITLE
First test targeting duckduckgo.com

### PR DESCRIPTION
**Pull request by bruklinas**

### CODE REVIEW

File: playwright.config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// @ts-check
+
+/** @type {import('@playwright/test').PlaywrightTestConfig} */
+const config = {
+  use: {
+    headless: true,
+    viewport: { width: 1920, height: 1080 },
+    ignoreHTTPSErrors: true,
+    video: "on-first-retry",
```
Comment (jevgenijusmarinuskinas) [inline playwright.config.js:9]: What was the reason for including this parameter?

**Review by jevgenijusmarinuskinas — APPROVED** [review]

File: playwright.config.js (hunk 1 above)
Comment (bruklinas) [reply playwright.config.js:9]: I'm not sure which parameter are you talking about :) I just copy-pasted this block from one of the playwright.dev documentation pages and changed headless to true for faster execution.
What exactly do you suggest? Should I remove/change any of the parameters? 
Thanks!

Comment (jevgenijusmarinuskinas) [reply playwright.config.js:9]: I was speaking about a video parameter. You can leave it or remove for the time being.

Comment (bruklinas) [reply playwright.config.js:9]: I'll remove with Testcase2 code push. Thanks again!

**Pull request merged**